projects/dynamicforms/src/util/validators/default-form-validators.ts

```diff
@@ -10,6 +10,11 @@ export class DefaultFormValidators extends Validators {
   static passwordRegex = /^(?=.*?[A-Z])(?=.*?[0-9])(?=.*?[!@#$%^*()_\-+{};:.,]).{6,}$/;
   static phoneNumberRegex = /^(?:\d{6,20})?$/;
 
+  static required(control: AbstractControl) {
+    const valid = !control.value || control.value?.trim() === '' ? false : true;
```

Comment:
Shouldn't this check if control value exists instead of not existing?
Also, `!!control.value?.trim()` should be enough.

Reply:
It checks both cases. Works fine with `!!control.value?.trim()`.